integration/using-error/pages/unhandled.tsx

```diff
@@ -0,0 +1,21 @@
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+
+const QUERY = gql`
+  {
+    hire {
+      id
+      name
+    }
+  }
+`;
+
+const Index = () => (
+  <Query query={QUERY}>
+    {() => {
+      throw new Error('unhandled');
```

Comment:
does this have to contain a string "unhandled" ?

Reply:
was just to match what the test is checking for https://github.com/lfades/next-with-apollo/pull/138/files#diff-562e5b2dd2b4c49308b9c4adbf4466adR44